File: kotlin-spring/src/main/kotlin/org/openapitools/model/RequestTimeFilterPostcodes.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.RequestTimeFilterPostcodesArrivalSearch
import org.openapitools.model.RequestTimeFilterPostcodesDepartureSearch
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param departureSearches 
 * @param arrivalSearches 
 */
data class RequestTimeFilterPostcodes(

    @field:Valid
    @get:Size(max=10)
    @field:JsonProperty("departure_searches") val departureSearches: kotlin.collections.List<RequestTimeFilterPostcodesDepartureSearch>? = null,

    @field:Valid
    @get:Size(max=10)
    @field:JsonProperty("arrival_searches") val arrivalSearches: kotlin.collections.List<RequestTimeFilterPostcodesArrivalSearch>? = null
) {

}

File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/RequestTimeFilterPostcodes.scala
package org.openapitools.server.model


/**
 * @param departureSearches  for example: ''null''
 * @param arrivalSearches  for example: ''null''
*/
final case class RequestTimeFilterPostcodes (
  departureSearches: Option[Seq[RequestTimeFilterPostcodesDepartureSearch]],
  arrivalSearches: Option[Seq[RequestTimeFilterPostcodesArrivalSearch]]
)

